C-Plus-Plus/other/tower_of_hanoi.cpp

```diff
@@ -0,0 +1,27 @@
+// C++ recursive function to
+// solve tower of hanoi puzzle
+#include <bits/stdc++.h>
+using namespace std;
+
+void towerOfHanoi(int n, char from_rod,
+                  char to_rod, char aux_rod)
+{
+    if (n == 1)
+    {
+        cout << "Move disk 1 from rod " << from_rod <<
+             " to rod " << to_rod << endl;
+        return;
+    }
+    towerOfHanoi(n - 1, from_rod, aux_rod, to_rod);
+    cout << "Move disk " << n << " from rod " << from_rod <<
+         " to rod " << to_rod << endl;
+    towerOfHanoi(n - 1, aux_rod, to_rod, from_rod);
+}
+
+// Driver code
+int main()
+{
+    int n = 4; // Number of disks

```

Comment:
Take Dynamic input and display suitable message before & after it

Reply:
Done!!